RELEASING.md

```diff
@@ -0,0 +1,24 @@
+# Releasing new version
+
+## Step-by-step
+
+In order to release a new version, there are several steps that must happen:
+
+ 0. make sure all issues (in GitHub and in JIRA) targeted to the version are resolved
+ 1. create a pull request in which you change the [version string](https://github.com/modcluster/mod_proxy_cluster/blob/main/native/include/mod_proxy_cluster.h#L21) according to the format described in [another section](#version-naming-conventions)
+ 2. get review and merge the pull request
+ 3. release the version through GH release functionality with the tag corresponding to version from the pull request
+ 4. mark the corresponding version released in [JIRA](https://issues.redhat.com/projects/MODCLUSTER?release-page) as well
```

Comment:
We need to define what the names will look like in Jira (we are currently using prefix native-1.2.3.Final)

Reply:
I think we're good using it in the future as well, but I'll add a note. Good catch.